dotty-tensorflow/src/main/scala/me/shadaj/scalapy/tensorflow/api/scalaUtils/ContextManager.scala

```diff
@@ -0,0 +1,21 @@
+package me.shadaj.scalapy.tensorflow.api.scalaUtils
+
+object ContextManager {
```

Comment:
Since at this level of abstraction we can/should turn away from the underlying Python mechanics/naming... I'd use some more telling name than `ContextManager`... ofc that's the official Python name for this entity, but it's also IMO one of the worst (least telling) names one could possibly imagine for a class :rofl:

Reply:
Ehhhh here we go again. 
OK, I don't have strong enough opinions to argue, so give me a name and I will put it here.

Reply:
As discussed... sth along the lines of `closeable resource manager` would be better